### DISCUSSION TITLE
Support for pages saved to the iOS home screen (PWAs)?

### DISCUSSION THREAD

**Discussion by ppixiv** [quoid/userscripts#347] [General]
Is it possible to support pages saved to the home screen on iOS?  My script is for a site for publishing illustrations, and it works much better when run this way since it can be fullscreen, which is much better for viewing illustrations and allows things like slideshows (screensaver/picture frame mode).  Safari's requestFullscreen doesn't work for this, since it doesn't overlap the status bar and puts a big nasty close button over the page.

I'm guessing it's just not supported by iOS, but thought I'd ask.  (I didn't even know GitHub had support for mini-forums.  It's nice to have a place to ask questions without opening a ticket.)

**Comment by ppixiv** [comment]
(I guess Safari does go fully fullscreen when the layout is correct--but that close button is a killer)

**Comment by quoid** [comment]
I don't think Safari has extension support for PWAs, yet - there was [an issue](https://github.com/quoid/userscripts/issues/251) about this that I just recently closed

**Comment by ppixiv** [comment]
Maybe some day.  I searched around a bit but only found somebody asking about it on SO, but no responses.
